### ISSUE TITLE
MQ-7 operating voltage

### ISSUE THREAD

**Issue by bill-orange** [swatish17/MQ7-Library#5]
Can the MQ-7 sensor run accurately on 3.3 volts? I would like to use this library with an ESP8266 and running the sensor on 5v would be difficult.

**Comment by swatish17** [comment]
Yes, you follow the instructions in the Readme.md file, and set the input voltage to 3.3 VDC.
